.github/workflows/multi-arch_compliance_operator_build.yml

```diff
@@ -0,0 +1,80 @@
+name: Multiarch build
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v*
```

Comment:
@rhmdnd the other workflow has  `v**`, is that necessary here also?

Reply:
@mrogers950 I have updated code as per other Workflwo e.g. I have did change of tag from v* to v** can you please revie it once.